src/main/resources/META-INF/rewrite/apache-maven-devcenter.yml

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright 2025 the original author or authors.
+# <p>
+# Licensed under the Moderne Source Available License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://docs.moderne.io/licensing/moderne-source-available-license
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+# An example DevCenter tracking Parent POM versions, illustrated by using the Apache Maven parent POM
+type: specs.openrewrite.org/v1beta/recipe
+name: io.moderne.devcenter.ApacheMavenDevCenter
+displayName: DevCenter
+description: >-
+  A DevCenter that tracks the latest Apache Maven parent POM versions and applies best practices.
+  This DevCenter includes recipes to upgrade the parent POMs of Apache Maven, as well as a collection of
+  best practices for Maven POMs.
+recipeList:
+  - io.moderne.devcenter.ParentPomUpgrade:
+      cardName: Move to the latest `maven-parent`
+      groupIdPattern: org.apache.maven
+      artifactIdPattern: maven-parent
+      version: 45
```

Comment:
I initially tried using `version: x` here, but that lead to all versions being completed, whereas it'd have been nice not to have to pin and update versions here. 🤷🏻